guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -399,21 +399,27 @@ When you're happy with the code on your computer, you need to commit the changes
 $ git commit -a
 ```
 
-At this point, your editor should be fired up and you can write a message for this commit. Well formatted and descriptive commit messages are extremely helpful for the others, especially when figuring out why given change was made, so please take the time to write it.
+This should open a buffer in your editor for writing a commit message (when you
```

Comment:
Thanks for reviewing! Would 'file' or 'window' be better?

The change in this paragraph addressed 2 things:

1) the current version is ambiguous for someone who might be doing this for the first time: whether the editor should already be fired up or if it is the git command that fires it up, and what the user should do to continue.

2) The importance of writing a good commit message is a different topic than using the editor and deserves its own paragraph.